nss-day-cohort-23/group-project-movie-history-a
# Prevent duplicate movies from printing

- Switched from .forEach loop to for loop so I could grab the index of the matching movie
- Used .splice to remove the matching movie from the global databaseMovies array 
- Switched the print order: your movies should print first, db movies should print last

How to test:
- When you search for a movie you've already added to your watch list, you should see your movie first (with stars)
- You should see movies from the database next, but no duplicates
